src/MetaModels/Filter/Setting/CustomSql.php

```diff
@@ -302,7 +303,6 @@ public function convertParameter($arrMatch)
         if ($index != $count || $var === null) {
             if (isset($arrArgs['default'])) {
                 $this->addParameter($arrArgs['default']);
-
```

Comment:
Can't be, we have the exact same on line [90](https://github.com/MetaModels/core/pull/987/files#diff-e71a8be0bf75328a5ff5718bd3414695R90), [168](https://github.com/MetaModels/core/pull/987/files#diff-e71a8be0bf75328a5ff5718bd3414695R168), [271](https://github.com/MetaModels/core/pull/987/files#diff-e71a8be0bf75328a5ff5718bd3414695R271), [315](https://github.com/MetaModels/core/pull/987/files#diff-e71a8be0bf75328a5ff5718bd3414695R315), [365](https://github.com/MetaModels/core/pull/987/files#diff-e71a8be0bf75328a5ff5718bd3414695R365).
However, the line break was correct here.